app/assets/stylesheets/cards.scss

```diff
@@ -0,0 +1,14 @@
+.cards {
+  list-style: none;
+
+  &__item {
+    float: left;
```

Comment:
Properties should be ordered border, border-radius, box-sizing, float, margin-right, padding, width